shell/browser/api/electron_api_tray.cc

```diff
@@ -72,7 +74,7 @@ gin::Handle<Tray> Tray::New(gin_helper::ErrorThrower thrower,
 
 #if BUILDFLAG(IS_WIN)
```

Comment:
Looks like we're only checking this in Windows, but the parameter is typed as a UUID regardless. I think that means on Mac we'd get  a null value for a non-GUID string even though it sounds like Cocoa would allow a non-GUID identifier.

Reply:
> Cocoa would allow a non-GUID

Cocoa might, but IMO for ease of API across platforms it should just be a GUID on both. Then folks can even use the _same_ GUID which is nice. Saying "this is a GUID on windows and on macOS can be any string have fun" just feels bad. So GUID everywhere is right IMO

Reply:
I'm in agreement with @MarshallOfSound here - unless there's a strong use case for non-guid identifiers we can point to on macOS, platform parity and consistency should take precedence here. That'll also make it easier to maintain and reduce platform specific parsing etc we need to do.